### ISSUE TITLE
Namespace in v0.6

### ISSUE THREAD

**Issue by gregsdennis** [beto-rodriguez/Live-Charts#73]
In the v0.6 notes, you say that you changed the namespace from `LiveCharts` to `lvc` to make things simpler in XAML.  While this is a noble effort, the namespace shouldn't be compromised to support another technology.  It should be descriptive of the code it contains.

This is why XML (and by extension, XAML) has namespace aliases.

    xmlns:lvc="clr-namespace:LiveCharts;assembly=LiveCharts"

This allows your client to make the decision to shorten it to `lvc` (or whatever abbreviation they choose) or leave `LiveCharts` if it's more meaningful for them.

In short, make your namespace meaningful, and allow your clients to shorten it if they choose.

All that said, it doesn't appear possible (with out-of-the-box .Net) to declare a default alias for a namespace.  I'll open an issue with Resharper as an enhancement to see if they can work out a way to do it.

**Comment by beto-rodriguez** [comment]
you are right, I will drop this down

**Comment by beto-rodriguez** [comment]
This is already droped down some commits ago

**Comment by gregsdennis** [comment]
For completeness: https://youtrack.jetbrains.com/issue/RSRP-453226?preventRedirect=true
